solidjs-tailwind/src/components/UserProfile/OrgList.jsx

```diff
@@ -0,0 +1,19 @@
+import { For } from 'solid-js';
+function OrgList(props) {
```

Comment:
Use splitProps as it shows a major feature in SolidJs

Reply:
@hdJerry I don't understand this comment. Can you explain why `splitProps` is required and how what you mean by ` it shows a major feature in SolidJs`?

Reply:
@hdJerry `splitProps` is used when you want to pick out certain properties from an object and pass the rest into a child component. It works the same with `Pick` in typescript.

It doesn't make sense to use it when all the properties passed are going to be used in that component.

If we need to show how major a feature it is, we can have a store that returns both user and repositories data in one object, then split it into two and pass it into their respective components(or into components where the data is needed). Or do the split inside the component and retrieve just the properties needed by the component.

Reply:
If the props are not reactive then no need. @vyktoremario